Assets/Scripts/ParticleSystemRenderer.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class ParticleSystemRenderer : MonoBehaviour {
```

Comment:
can you go to the particle system itself, go to the renderer, and change the layer there?